.swiftlint.yml

```diff
@@ -8,7 +8,7 @@ disabled_rules:
   - opening_brace
   - todo
 
-line_length: 100
+line_length: 120
```

Comment:
I find that the limit of 100 is too narrow in practice, there are some comments with links that don't fit.